### ISSUE TITLE
facetool settings are not written to/read from PNG files

### ISSUE THREAD

**Issue by db3000** [invoke-ai/InvokeAI#1101]
**Describe your environment**
- GPU: [cuda/amd/mps/cpu]
- VRAM: 12gb
- CPU arch: x86
- OS: Linux
- Python: miniconda
- Branch: main/development
- Commit: 32dab7d4bf4f56121973fdb325651a4c6efc94f9

**Describe the bug**
The facetool parameters (`-G`, `-ft`, `-cf`) are not saved in the PNG output, so `!fetch` can not load it. Instead `!fetch` puts in the default for these parameters. I noticed this while looking into #1094, but I don't think it is related to #1078 . In the main branch the values are not saved either, but at least the defaults are not output when running `!fetch`

**To Reproduce**
Run codeformer with custom parameters on an image and then `!fetch` it, which results in default parameters and gfpgan selected.
```
invoke> test -s1 -ft codeformer -G 1.1 -cf 0.9
[59] outputs/img-samples/000049.628396732.png: "test" -s 1 -S 628396732 -W 512 -H 512 -C 7.5 -A k_lms -ft codeformer -G 1.1 -cf 0.9

invoke> !fetch outputs/img-samples/000049.628396732.png
invoke> "test:1.0" -s 1 -S 628396732 -W 512 -H 512 -C 7.5 -A k_lms -ft gfpgan -cf 0.75
```

On main branch:
```
invoke> test -s1 -ft codeformer -G 1.1 -cf 0.9
[60] outputs/img-samples/000050.1511997885.png: "test" -s 1 -S 1511997885 -W 512 -H 512 -C 7.5 -A k_lms -G 1.1

invoke> !fetch outputs/img-samples/000050.1511997885.png
invoke> "test:1.0" -s 1 -S 1511997885 -W 512 -H 512 -C 7.5 -A k_lms
```

**Expected behavior**
The facetool parameters should be saved, or should be loaded from the `postprocessing` section of the metadata. Alternatively, no facetool parameters should be output when using `!fetch`

**Additional context**
Followup to https://github.com/invoke-ai/InvokeAI/issues/1094#issuecomment-1279014269

**Comment by lstein** [comment]
Yeah, the metadata reading/writing of postprocessing args needs to be fixed. I'll clean it up over the weekend.

**Comment by psychedelicious** [comment]
Fixed by #1119
